genesiscommunitysuccess/custom-elements-lsp
# feat(custom-elements-lsp): pass user config into any plugins FUI-1600

🤔  &nbsp; **What does this PR do?**

- Plugins for the CEP now will have the user config from the tsconfig passed in during initialisation

📑  &nbsp; **How should this be tested?**

Update test steps to match your PR:

```
1. Checkout branch
2. `npm run bootstrap`
3. `npm run test`

Other packages are completely tested via tests from `npm run test`. To test the LSP is working correctly in VSCode:
1. `cd packages/showcase/example`
2. `code .` (or manually open VSCode via the UI to the `example` directory on your filesystem).
3. Open a typescript file such as `root.ts`
4. In the Command Palette choose `TypeScript: select typescript version...` and then choose the workspace version
5. The LSP should be enabled. You will need to make a code change after the LSP is enabled before you see any Intellisense.
```

Everything should work as before. No functionality should have changed.

✅  &nbsp; **Checklist**



- [ ] I have added tests for my changes. - N/A
